css/style.css

```diff
@@ -0,0 +1,101 @@
+body {
+  background-color: darkslateblue;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 14px;
+  line-height: 1.5;
+  color: whitesmoke;
+  margin: 0;
+  padding: 0;
+}
+
+body::before {
+  content: '';
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
+  z-index: -1;
+  background-image: url(https://img5.goodfon.com/wallpaper/nbig/f/e1/tsvet-forma-fon-230.jpg);
+  background-position: center;
+}
+
+#title {
+  font-size: 2.5rem;
+  font-weight: 400;
+  line-height: 1.5;
+}
+
+#title,
+#description {
+  text-align: center;
+}
+
+h1 {
+  font-weight: 400;
+  line-height: 1.2;
+}
+
+p {
+  font-size: 1.2rem;
+}
+
+h1,
+p {
+  margin-top: 0;
+  margin-bottom: 0.5rem;
+}
```

Comment:
There may be occasions when two or more statements apply to the same element. It is also possible that there may be a conflict between them.

Remember that CSS is declared in cascading style, so it will override previous declarations for the same elements.

Reply:
- [ ] kindly consider making your suggested change clear and easy to understand cause currently someone can't really know what you are suggesting as it is not added  check on the below 👇 example on how a suggestion is clear 

- This is a clear comment: The README's "Built With" and "Getting Started" sections are currently empty. Why don't you add content, listing the technology used in the project and how to run it, to them so your README is more informative and professional?

- This is a vague comment: The README is not descriptive, why don't you make it better?